src/App.css

```diff
@@ -9,21 +9,72 @@ body {
 
 .reveal {
   position: relative;
-  transform: translateY(120px);
+  transform: translateY(200px);
   opacity: 0;
-  transition: 0.6s all ease;
+  transition: 1s all ease;
+}
+.reveal.active {
+  transform: translateY(0);
+  opacity: 1;
 }
 
+.reveal-img {
+  position: relative;
+  transform: translateX(-200px);
+  opacity: 0;
+  transition: 1s all ease;
+}
 
-.reveal-slow {
-  transition: 1.5s all ease !important;
+.reveal-img.done {
+  transform: translateX(0);
+  opacity: 1;
 }
 
-.reveal.active {
+.reveal-program {
+  position: relative;
+  transform: translateY(50px);
+  opacity: 0;
+  transition: 1s all ease;
+}
+.reveal-program.done {
   transform: translateY(0);
   opacity: 1;
 }
 
+.reveal-left {
+  position: relative;
+  transform: translateX(-10vw);
+  opacity: 0;
+  transition: 1s all ease;
+}
+
+.reveal-left.done {
+  transform: translateX(0);
+  opacity: 1;
+}
+
+.reveal-right {
+  position: relative;
+  transform: translateX(10vw);
+  opacity: 0;
+  transition: 1s all ease;
+}
+
+.reveal-right.done {
+  transform: translateX(0);
+  opacity: 1;
+}
+
+.reveal-slow {
+  transition: 1.5s all ease !important;
```

Comment:
As @tomesfilip  pointed earlier, is important! necesarry?

Reply:
Here not, sorry...